cpp/formatting/snippets/one-parameter-per-line-if-they-do-no-fit.cpp

```diff
@@ -0,0 +1,10 @@
+// One parameter per line if they don't all fit
+// clang-format: AllowAllParametersOfDeclarationOnNextLine: false
+// clang-format: BinPackParameters: False
+
+void andAVeryLongFunctionName(the_long_parameter1,
+                              the_long_parameter2,
+                              the_long_parameter3,
+                              the_long_parameter4);
+
+void andALongFunctionName(the_long_parameter1, the_long_parameter2);
```

Comment:
I prefer this!

Reply:
`the_long_parameter1` is not a long parameter.
From a real world thing :)
```
HighFive::DataSet dump(HighFive::File& file,
                       const std::string& path,
                       const std::vector<T>& data,
                       HighFive::Mode mode = HighFive::Mode::Create);
```
Anyway, I totally agree with the change (I even had suggested that in the gerrit review)

Reply:
I changed with your HighFive example.